Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -582,8 +582,13 @@ class KalmanFitter {
     Result<void> filter(const Surface* surface, propagator_state_t& state,
                         const stepper_t& stepper, const navigator_t& navigator,
                         result_type& result) const {
+      const bool precedingMeasurementExists = (result.measurementStates > 0);
+      const bool surfaceIsSensitive =
+          (surface->associatedDetectorElement() != nullptr);
+      const bool surfaceHasMaterial = (surface->surfaceMaterial() != nullptr);
+      const auto sourcelink_it = inputMeasurements->find(surface->geometryId());
```

Comment:
```suggestion
      const auto sourcelinkIt = inputMeasurements->find(surface->geometryId());
```

Reply:
did we converge on `sourcelink` vs `sourceLink`?

Reply:
I think we didn't converge at all. I now unified the naming:
- https://github.com/acts-project/acts/pull/3647